internal/controller/rsm/transformer_status.go

```diff
@@ -76,4 +76,4 @@ func (t *CSSetStatusTransformer) Transform(ctx graph.TransformContext, dag *grap
 	return nil
 }
 
-var _ graph.Transformer = &CSSetStatusTransformer{}
+var _ graph.Transformer = &ObjectStatusTransformer{}
```

Comment:
similar comments for placement, check them all.

Reply:
done: [ed95314](https://github.com/apecloud/kubeblocks/pull/3733/commits/ed9531499b9264b9d310314233839f85bc09ba6b)